src/app/containers/ConsentBanner/Canonical/logic/README.md

```diff
@@ -0,0 +1,45 @@
+# Canonical Cookie Banner Logic
+
+```js
+import logic from './index';
+
+const {
+    runInitial,
+    privacyOnAllow,
+    privacyOnReject,
+    cookieOnAllow,
+    cookieOnReject,
+  } = logic({ showPrivacyBanner, showCookieBanner });
```

Comment:
What's the purpose of having these arguments ` showPrivacyBanner`, `showCookieBanner` as functions rather than booleans?

It'd be good to fix the API of the package to be easily transferrable into Psammead, as an npm package. e.g. 
```
import {
    runInitial,
    privacyOnAllow,
    privacyOnReject,
    cookieOnAllow,
    cookieOnReject,
} from '.';
```

Reply:
Theyre functions as thats whats used by useState. I could make them booleans but I would need to watch the value, then update the state with a function, which seems a bit messy IMO 🙈

Reply:
Okay. That would assume that all users of this library would want to use React's `useState`, which isn't necessarily the case. 

It would be good to explain under `input functions` section below why this library is not providing the `showPrivacyBanner` `showCookieBanner` functions. 
Also, whether these functions should be carrying out the `display: none; visibility: hidden;` aspect of hiding/showing the front-end components. 
Currently, the expected behaviour of these input functions is not clear.

Reply:
>Okay. That would assume that all users of this library would want to use React's useState, which isn't necessarily the case.

I kinda disagree, anyone can write a function to show a banner, doesn't need to be react.

Ile update the doc for the rest :P

Reply:
done :)